### PR TITLE
[fix](statistics) fix not master fe first query cost long time

### PR DESCRIPTION
**Pull request by ryanzryu**
## Proposed changes
call StatisticsCache.preHeat() for transferToNonMaster. then the not master fe will cache all table's statistics. so that first query will cost a normal time.
Issue Number: #26738

### CODE REVIEW

**Review by Kikyou1997 — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java [hunk 1]
```diff
@@ -1559,6 +1559,10 @@ private void transferToNonMaster(FrontendNodeType newType) {
 
         MetricRepo.init();
 
+        if (analysisManager != null) {
+            analysisManager.getStatisticsCache().preHeat();
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java:1563]: transferToNonMaster is not only be called when fe start, but also when master changed. Could we find an appropriate place, let all fe do preHeat just when it start?

**Comment by morrySnow** [comment]
btw, u should open a PR for branch master first. and then pick it from master to branch-2.0 when master PR merged.

**Comment by ryanzryu** [comment]
> btw, u should open a PR for branch master first. and then pick it from master to branch-2.0 when master PR merged.
I found master already fix in this  pr https://github.com/apache/doris/pull/24237/files by Kikyou1997, this pr Mixed with other changes,and I want to only fix this question for 2.0 branch

**Review by morrySnow — APPROVED** [review]

**Comment by morrySnow** [comment]
run buildall

**Comment by ryanzryu** [comment]
您的邮件我已收到。

**Pull request merged**